Example/Tests/TSSwiftMarkdownParserTests.swift

```diff
@@ -205,9 +205,10 @@ class TSSwiftMarkdownParserTests: XCTestCase {
             attributedString.replaceCharactersInRange(range, withString: "    • ")
         }, textFormattingBlock: nil)
         
-        parser.addStrongParsingWithFormattingBlock { attributedString, range in
+        
```

Comment:
Undo this change. You don't need the ()s and you don't need the second empty line